### ISSUE TITLE
Colons in image urls don't work

### ISSUE THREAD

**Issue by tvynguyen** [ZiffDavis/laravel-eloquent-imagery#4]
We should think of a different format that doesn't include colons in the url

**Comment by ralphschindler** [comment]
Indeed, we originally punted on this and I recall we wanted to revisit the scheme.. here are some options we should consider:

```
current:
    182-hero-sm.fit:lpad_size:1000x1000_v:1539880411.png

(1) `.` op separator, `_` arg separator
    182-hero-sm.fit_lpad.size_1000x1000.v_1539880411.png
    (** arguments cannot have underscores)

(2) `.` op separator, `--` argument separator, 
    182-hero-sm.fit--lpad.size--1000x1000.v--1539880411.png

(3) `.` op separator, arguments in (), 
    182-hero-sm.fit(lpad).size(1000x1000).v(1539880411).png

(4) `,` op separator, `--` argument separator
    182-hero-sm.fit--lpad,size--1000x1000,v--1539880411.png

(5) `,` op separator, arguments in (), 
    182-hero-sm.fit(lpad),size(1000x1000),v(1539880411).png

(6) `.` op separator, `-` argument separator
    182-hero-sm.fit-lpad.size-1000x1000.v-15324324553.png
```

**Comment by ralphschindler** [comment]
Fixed in PR #5
